### ISSUE TITLE
Add basic smcl highlighting

### ISSUE THREAD

**Issue by kylebarron** [kylebarron/language-stata#62]

**Comment by adamrossnelson** [comment]
Good idea. I was going to submit this as an issue/request. Glad it was on the mind of others too. To my knowledge there isn't an existing editor that gives smcl highlighting.

**Comment by kylebarron** [comment]
I've never had to write in it, but it seems like there's clear tags for bold, italics, bullet points, and code, so it should be easy to color those parts

**Comment by kylebarron** [comment]
I'll just go through `help smcl` and add some basic rules. I want to focus more time right now on stata-exec than the language though.

**Comment by apoorvalal** [comment]
I rarely document my ados because of how tedious I find smcl writing. As an alternative route (and one that doesn't strictly belong on this repo), would it be possible to write a pandoc filter to do markdown <> smcl conversions? I think Sergio of `reghdfe` fame has written some pandoc filters, maybe that's an option?

**Comment by kylebarron** [comment]
You can start with https://github.com/michaelstepner/pandoc-smcl

**Comment by apoorvalal** [comment]
Wow, didn't know this existed. Cheers!
